components/home/features/Features.tsx

```diff
@@ -0,0 +1,72 @@
+import { Col, Container, Row } from "react-bootstrap";
+import { CloudWithCheckmark } from "./icons/CloudWithCheckmark";
+import { LineGraph } from "./icons/LineGraph";
+import { OpenBox } from "./icons/OpenBox";
+import { Puzzle } from "./icons/Puzzle";
+import { Sliders } from "./icons/Sliders";
+import { Speedometer } from "./icons/Speedometer";
+import styles from "./Features.module.scss";
+
+export const Features = () => {
+  return (
+    <Container className={`p-4 ${styles.container}`}>
```

Comment:
Something about the padding/spacing in this Container is different from the "Why Open Source?" section. I imagine they should be the same. I think the spacing of the "Why Open Source?" section is closer to correct. 

![2021-12-14_12-19-31](https://user-images.githubusercontent.com/633226/146073641-e09a5b08-4696-4822-a081-fb37749fda33.png)

![2021-12-14_12-20-34](https://user-images.githubusercontent.com/633226/146073668-676419d0-15b5-469c-a738-b51c19315380.png)

Reply:
These should be fixed. I've set mins on all of them.